generator/src/main.rs

```diff
@@ -1518,6 +1518,7 @@ impl Parser {
                             #(#ext_set_inits)*
                             bytes => {
                                 if let Ok(name) = std::str::from_utf8(bytes) {
+                                    // This is probably always true but we check just to be sure.
```

Comment:
It's not "probably" true, it's guaranteed by the spec. Any implementation failing to do this is buggy.